public_profiler/v3.5-DD/ogl/skills.js

```diff
@@ -391,6 +392,10 @@ var skillsHelpURL = Object.assign(
     // Wizard special abilities.
     'summon familiar':                      baseHelpURL + "classes/sorcererWizard.htm#wizardFamiliar",
     'scribe scroll':                        baseHelpURL + "classes/sorcererWizard.htm#scribeScroll",
+    'empathic link':                        baseHelpURL + "classes/sorcererWizard.htm#familiarEmpathicLink",
+    'scry on familiar':                     baseHelpURL + "classes/sorcererWizard.htm#scryonFamiliar", speakwithMaster
+    'speak with master':                    baseHelpURL + "classes/sorcererWizard.htm#speakwithMaster",
+    'speak with familiar':                  baseHelpURL + "classes/sorcererWizard.htm#speakwithMaster",
```

Comment:
Included this as an alias, since it's two-way communication and this is the form that (logically enough) appears on Aramil's character sheet.